kodi_addon_checker/check_dependencies.py

```diff
@@ -32,6 +35,11 @@ def check_addon_dependencies(report: Report, repo_addons: dict, parsed_xml, bran
             if required_addon not in ignore:
                 report.add(Record(
                     PROBLEM, "Required addon %s not available in current repository." % required_addon))
+            else:
+                version = VERSION_ATTRB[required_addon][branch_name]
+                if not LooseVersion(version) == LooseVersion(required_version):
+                    report.add(Record(WARNING, "You are using and older version xbmc.python"))
```

Comment:
2 things:
- Don't hardcode `xbmc.python` here since the idea is to make it independent of the addon
- Can you report what version the author should use (i.e. "For {branch} it is advised to set {addon} version to {required_version}.